public/v4/apps/outline.yml

```diff
@@ -64,6 +64,10 @@ services:
             SMTP_TLS_CIPHERS: $$cap_smtp_tls_ciphers
             SMTP_SECURE: $$cap_smtp_secure
             DEFAULT_LANGUAGE: $$cap_default_language
+        caproverExtra:
+            - FROM outlinewiki/outline:$$cap_outline_version
+            - EXPOSE 3000
+            - CMD yarn db:migrate --env production-ssl-disabled; yarn start
```

Comment:
Is this tested? It looks like the command here is different from the snippet below?  `db:migrate` vs `sequelize:migrate` - are these both the same?

Reply:
https://github.com/outline/outline/issues/4772
It seems sequelize is no longer available.

Reply:
Maybe `sequelize:migrate` is deprecated command, I followed error instruction to fix


![2023-02-25_17-10](https://user-images.githubusercontent.com/14988496/221377960-a954cae5-7c2c-434e-a6a2-0bea1b1b68cf.png)